content/en/remote-network-agent/overview.md

```diff
@@ -1,35 +1,50 @@
-  * Install using default configuration
-  * Not recommended for production environments
-* [**Helm**]({{< ref "remote-network-agent/install-agent-helm" >}})
-  * Install using default or advanced configuration
-  * Recommended for production environments
+To get started with CD-as-a-Service or when adding a new deployment target without an existing RNA, you need to install the RNA. Once installed, there is no need for further interaction with the RNA. CD-as-a-Service takes care of the rest, seamlessly handling your deployments.
 
-{{% alert title="Important" color="warning" %}}
-If you are coming to this guide from the UI **Install a Remote Network Agent** screen because you want to manually install the RNA, follow the [kubectl instructions]({{< ref "remote-network-agent/install-agent-kubectl" >}}) or the [Helm instructions]({{< ref "remote-network-agent/install-agent-helm" >}}). Use the cluster name you created to identify your RNA. Copy the **Client ID** and **Client Secret** from the UI.
+Please refer to the [Architecture](/architecture) page for more details on how RNA fits in with the rest of the CD-as-a-Service core components.
 
-_Do not close the pop-up window in the UI until you have completed RNA installation. The credentials in the pop-up window are deleted if you close the window before the RNA has connected._
-{{% /alert %}}
+## Core Features
+
+### Kubernetes Cluster Mode
+
+When installed into a Kubernetes cluster, RNA can register the cluster as a deployable target from within CD-as-a-Service.
+
+RNA is installed with a service account and its credentials are used by Armory CD-as-a-Service when orchestrating kubernetes deployments.
+
+See the [Installation Guide](/remote-network-agent/install) to get started and [Production Configuration and Use](/remote-network-agent/production-configuration-and-use) for advanced configuration, such as configuring the service account permissions or opting out of this mode.
+
+### Relaying Traffic to Private Networks
+
+When you install the agent configured with Armory credentials into your private network(s). The agent connects to Armory's Agent Hub registering the private network with an agent identifier.
+
+Agent Hub can route network traffic from internal authenticated Armory services to your privately networked 
+resources via a multi-target network relay that pipes data through encrypted gRPC tunnels to their destination through the remote network agent.
+
+<div style="display: flex; justify-content: center; padding-left: 20px; padding-right: 25px; padding-bottom: 15px">
+  <div style="display: flex; flex-flow: column; text-align: center">
+    <div>
+      <img src="/images/cdaas/rna-arch.png" alt="CD-as-a-Service Remote Network Agent Architecture Diagram" style="max-width: 1200px; width: 100%">
+    </div>
+    <b>Remote Network Agent securely integrates your on-prem tools with CD-as-a-Service.</b>
+  </div>
+</div>
```

Comment:
[Screencast from 07-12-2023 12:18:41 PM.webm](https://github.com/armory/docs-cdaas/assets/711726/5f70c2eb-a97e-42a6-a80a-fb236afba5f3)

I'm gonna try move this to a short code, I suspect it might be causing the build to fail.

Which is weird because it works fine locally see attached video.